### PR TITLE
Swift: Add RNCryptor sinks to swift/constant-password

### PR DESCRIPTION
**Pull request by geoffw0**
Add RNCryptor sinks to `swift/constant-password`.  This change allows the query to find two results in DVIA-v2 (DCA should confirm).

### CODE REVIEW

File: swift/ql/test/query-tests/Security/CWE-259/rncryptor.swift [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+	// all methods
+
+	let myKeyDerivationSettings = RNCryptorKeyDerivationSettings()
+	let myHandler = {}
+	let myIV = Data(0)
+	let mySalt = Data(0)
+	let mySalt2 = Data(0)
+
+	let _ = myEncryptor.key(forPassword: myConstPassword, salt: mySalt, settings: myKeyDerivationSettings) // BADx
```
Comment (MathiasVP) [inline swift/ql/test/query-tests/Security/CWE-259/rncryptor.swift:91]: Right?
```suggestion
	let _ = myEncryptor.key(forPassword: myConstPassword, salt: mySalt, settings: myKeyDerivationSettings) // BAD
```

Comment (geoffw0) [reply swift/ql/test/query-tests/Security/CWE-259/rncryptor.swift:91]: I blame the slightly smaller keyboard on my Mac laptop. :P

**Review by MathiasVP — APPROVED** [review]
LGTM once DCA is happy!

**Comment by geoffw0** [comment]
DCA is happy, and shows the two new alerts. :tada:

They are listed as 'unclassified', which means classification failed.  I think that's just because they were tagged as `swift/hardcoded-key` rather than `swift/constant-password`.  I will fix that today...

**Comment by geoffw0** [comment]
> I will fix that today...

... but the fix will be in the DCA repo.  So I might as well merge this PR now.

**Pull request merged**